libs/agora/shared/src/lib/loading-overlay/loading-overlay.component.ts

```diff
@@ -18,9 +18,11 @@ export class LoadingOverlayComponent implements OnInit {
   @Input() isActive = false;
 
   ngOnInit() {
+    console.log('global', this.isGlobal);
     if (this.isGlobal) {
       this.helperService.loadingChange.subscribe(() => {
         this.isActive = this.helperService.getLoading();
+        console.log('active', this.isActive);
```

Comment:
Could the log statements be removed? 

```suggestion
    if (this.isGlobal) {
      this.helperService.loadingChange.subscribe(() => {
        this.isActive = this.helperService.getLoading();
```

Reply:
This component deleted as it is no longer needed.